### PR TITLE
Loading component

### LINKED ISSUE #52: Loading Component
- [x] Create a Loading.jsx component
   - [x] Its content is wrapped around a "container" div 
   - [x] Has a content div with a  class of "content"
      - [x] Renders a "Finding you a match..." H1
- [x] Proper Formatting
- [x] Simple styling

### PR DESCRIPTION
**Pull request by gabremo**
Implementing a Loading Modal to let the user know they're in a queue.

### CODE REVIEW

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx [hunk 1]
```diff
@@ -62,15 +69,17 @@ export default function ProfilePage() {
 
     socket.on('redirectToRoom', (roomURL) => {
       console.log(roomURL)
-        // redirect to new URL
-        window.location = roomURL;
+      setIsLoading(false);
+      // redirect to new URL
+      window.location = roomURL;
+      console.log(isLoading)
```
Comment (LifeBringer) [inline capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Needs to be removed

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx [hunk 2]
```diff
@@ -62,15 +69,17 @@ export default function ProfilePage() {
 
     socket.on('redirectToRoom', (roomURL) => {
       console.log(roomURL)
```
Comment (LifeBringer) [inline capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Needs to be removed

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Console.log

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx (hunk 1 above)
Comment (gabremo) [reply capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Done!

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx (hunk 2 above)
Comment (gabremo) [reply capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Got it.

**Review by LifeBringer — APPROVED** [review]
LGTM

**Pull request merged**